### PR TITLE
Add mango coconut rice recipe

### PR DESCRIPTION
**Pull request by NicMoe**

### CODE REVIEW

File: desserts/mango-coconut-rice.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Mango coconut rice
+
+This Thai dessert compliments salty and spicy food well. This recipe is inspired by a Food Network [mango coconut rice recipe](https://www.foodnetwork.com/recipes/mango-coconut-rice-recipe-1937791) from Paula Dean and a [caramelized mango recipe](https://www.myrecipes.com/recipe/caramelized-mangoes) from MyRecipes.
+
+
+## Ingredients
+
+- 1 tablespoon olive oil
+- 1.5 cups long-grain rice (ideally sushi rice)
```
Comment (diego-codes) [inline desserts/mango-coconut-rice.md]: Sushi rice is a type of short-grain rice. Which is it then?

File: desserts/mango-coconut-rice.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Mango coconut rice
+
+This Thai dessert compliments salty and spicy food well. This recipe is inspired by a Food Network [mango coconut rice recipe](https://www.foodnetwork.com/recipes/mango-coconut-rice-recipe-1937791) from Paula Dean and a [caramelized mango recipe](https://www.myrecipes.com/recipe/caramelized-mangoes) from MyRecipes.
+
+
+## Ingredients
+
+- 1 tablespoon olive oil
```
Comment (diego-codes) [inline desserts/mango-coconut-rice.md]: Should these measurements use abbreviations? It is how we have other recipes here.

Comment (diego-codes) [inline desserts/mango-coconut-rice.md]: I love olive oil, but is it really the right type of oil for this? It's too flavorful IMO. Also, Thai food would definitely not use olive oil.

File: desserts/mango-coconut-rice.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- 1 large, ripe mango, peeled and cubed (about 2 cups)
+
+
+## Instructions
+
+1. Peel the ripe mango and cut into cubes, about 1/3 to 1/2 inch per side.
+
+2. Combine 2 tablespoons sugar and 1.5 tablespoons water in a medium pan over medium-high heat; cook until sugar dissolves. Continue cooking 3 minutes or until golden; do not stir. Reduce heat to medium.
+
+3. Add cubed mango to pan, tossing gently. Cook 10 minutes or until mango is lightly browned, stirring frequently.
```
Comment (diego-codes) [inline desserts/mango-coconut-rice.md]: So my idea of caramelizing mangoes would only apply to frozen and unripe mango. When they're ripe they are already sweet and probably don't need any more sugar.

**Review by diego-codes — COMMENTED** [review]
I have questions

File: desserts/mango-coconut-rice.md (hunk 2 above)
Comment (NicMoe) [inline desserts/mango-coconut-rice.md]:
```suggestion
- 1 tbsp olive oil
```

File: desserts/mango-coconut-rice.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- 2 tbsp sugar
+
+
+## Instructions
+
+1. Peel the ripe mango and cut into cubes, about 1/3 to 1/2 inch per side.
+
+2. (If you are using frozen mango) Combine 2 tablespoons sugar and 1.5 tablespoons water in a medium pan over medium-high heat; cook until sugar dissolves. Continue cooking 3 minutes or until golden; do not stir. Reduce heat to medium. Add cubed mango to pan, tossing gently. Cook 10 minutes or until mango is lightly browned, stirring frequently.
+
+3. In another large saucepan, heat the tablespoon of olive oil over medium-high heat. Stir in the 1.5 cups of rice to coat with the oil.
```
Comment (diego-codes) [inline desserts/mango-coconut-rice.md]:
```suggestion
3. In another large saucepan, heat the tablespoon of oil over medium-high heat. Stir in the 1.5 cups of rice to coat with the oil.
```

File: desserts/mango-coconut-rice.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+4. Add the can of coconut milk, 1/2 cup water, and teaspoon of salt; bring to a boil. Stir in the caramelized mango.
```
Comment (diego-codes) [inline desserts/mango-coconut-rice.md]:
```suggestion
4. Add the can of coconut milk, 1/2 cup water, and teaspoon of salt; bring to a boil. Stir in the mango.
```

**Review by diego-codes — CHANGES_REQUESTED** [review]

**Comment by NicMoe** [comment]
@diego-codes please review and merge if good.

**Review by diego-codes — APPROVED** [review]

**Pull request merged**